clickhouse/datadog_checks/clickhouse/queries.py

```diff
@@ -3,10 +3,32 @@
 # Licensed under a 3-clause BSD style license (see LICENSE)
 from .utils import compact_query
 
+class QueryBuilder(object):
+    def get_queries(self, cluster_name):
+        queries = [
+                queries.SystemMetrics,
+                queries.SystemEvents,
+                queries.SystemAsynchronousMetrics,
+                queries.SystemParts,
+                queries.SystemReplicas,
+                queries.SystemDictionaries,
+            ]
+        if not (cluster_name and cluster_name.strip()):
+            return queries
+        else: 
+            updatedQueries = []
+            for q in queries:
+                nq = q
```

Comment:
Can you use more specific variable names other than `nq` and `q`? I am not sure what `nq` is supposed to be.

Reply:
sure, will update